stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/response/BannedUserResponse.kt

```diff
@@ -12,6 +12,6 @@ internal data class BannedUserResponse(
     val channel: DownstreamChannelDto?,
     val created_at: Date?,
     val expires: Date?,
-    val shadow: Boolean,
+    val shadow: Boolean?,
```

Comment:
If the value is occasionally missing but is never literally `null`, we could do `= false` here to get a default value in place.